### ISSUE TITLE
[A11y] - There is no focus style on Language Switcher Dropdown

### ISSUE THREAD

**Issue by carcam** [joomla/joomla-cms#34400]
It seems that language switcher dropdown does not have a different style when receiving the focus. This fails with https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-minimum .

### Steps to reproduce the issue

1. Add Language Switcher to the site
2. Go to the homepage of the site
3. Use your keyboard (Tab key) to reach the dropdown

### Expected result

When reaching the dropdown it should show a focus state like in the following screenshot:

![Captura de pantalla 2021-06-04 a las 16 54 22](https://user-images.githubusercontent.com/1695138/120821439-c0a71700-c555-11eb-8322-d0bbc0d35829.png)

### Actual result

The dropdown shows no change when having the focus as per the screenshot:

![Captura de pantalla 2021-06-04 a las 16 53 38](https://user-images.githubusercontent.com/1695138/120821341-aa00c000-c555-11eb-8786-b72a8c84e5fc.png)

### System information (as much as possible)

- Joomla 4 from GH repo after npm ci and composer update.

### Additional comments

This seems to be a general issue when using BS5 dropdown so probably we should consider adding a generic fix like:

`a.btn.dropdown-toggle:focus {
    border: 1px solid #8894AA;
}`

@joomla/joomla-accessibility-team-jat

**Comment by brianteeman** [comment]
confirm the issue but not the fix - the problem is the outline that is being set to 0 here

```
  .btn-check:focus + &,
  &:focus {
    outline: 0;
    box-shadow: $btn-focus-box-shadow;
  }

```

**Comment by brianteeman** [comment]
https://github.com/joomla/joomla-cms/pull/33929#issuecomment-842982994

**Comment by drmenzelit** [comment]
@carcam can you please test #33929 and let me know what is missing?

**Comment by carcam** [comment]
> @carcam can you please test #33929 and let me know what is missing?

Great job there!! That fixes this issue among others we had with this switcher ;)



> confirm the issue but not the fix - the problem is the outline that is being set to 0 here
> 
> ```
>   .btn-check:focus + &,
>   &:focus {
>     outline: 0;
>     box-shadow: $btn-focus-box-shadow;
>   }
> ```

I agree, the outline is much better.

Thanks @brianteeman and @drmenzelit !!

**Comment by richard67** [comment]
Closing as having a pull request. Please test #33929 . Thanks in advance.
